ts/drag-drop-touch.ts

```diff
@@ -108,13 +108,7 @@ class DragDropTouch {
   private _img: HTMLElement | null;
   private _imgCustom: HTMLElement | null;
   private _imgOffset: Point;
-
-  // Note that this typing is _not_ true for Node, because it does not follow
-  // the official JS Timers specification. However, Node does not have native
-  // support for typescript, and so by the time this code runs there is no
-  // problem. Similarly, native-typescript engines like Deno or Bun *do* follow
-  // the spec and so the following typing is correct when running in those.
-  private _pressHoldIntervalId?: number | undefined;
+  private _pressHoldIntervalId?: ReturnType<typeof setTimeout> | undefined;
```

Comment:
A handy way to circumvent "this could be Node, or could be the browser, we don't know" =)